packages/editor-ui/src/store.ts

```diff
@@ -731,6 +731,11 @@ export const store = new Vuex.Store({
 			return state.activeWorkflows;
 		},
 
+		getActiveWorkflowTriggerNodes: (state, getters) => {
```

Comment:
nitpick - I know I suggested this name but `activeWorkflowTriggerNodes` might be more appropriate. I like to use `get` in names when it's returning a function

Reply:
I changed the getter to activeWorkflowTriggerNodes